### PR TITLE
e2e test improvements: run "kubemci list" and more error checking

### PR DESCRIPTION
**Pull request by G-Harmon**
reviewers: @nikhiljindal 
cc: @csbell @madhusudancs 

Nikhil, I know you have a PR out for the e2e test now as well. It might be easier if this one goes in first? They both touch a lot of this file.

---
This change is [Reviewable](https://reviewable.io/reviews/googlecloudplatform/k8s-multicluster-ingress/103)

### CODE REVIEW

**Comment by nikhiljindal** [comment]
ok I will wait to review this one before merging the other one

File: test/e2e/cases/basic.go [hunk 1]
```diff
@@ -16,13 +16,17 @@ package e2e
...
+	"strings"
 	"time"
 
 	"github.com/golang/glog"
 
 	"github.com/GoogleCloudPlatform/k8s-multicluster-ingress/app/kubemci/pkg/kubeutils"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
 )
 
+var kubemci_bin = "./kubemci"
```
Comment (nikhiljindal) [inline test/e2e/cases/basic.go]: I dont think we should do this.
kubemci in PATH should be enough?

File: test/e2e/cases/basic.go [hunk 2]
```diff
@@ -16,13 +16,17 @@ package e2e
 
 import (
 	"fmt"
+	"strings"
 	"time"
 
 	"github.com/golang/glog"
 
 	"github.com/GoogleCloudPlatform/k8s-multicluster-ingress/app/kubemci/pkg/kubeutils"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
```
Comment (nikhiljindal) [inline test/e2e/cases/basic.go]: We really need to find out why I dont require this and you do :)
Is it just me who doesnt require it or is it just you who requires this :)

File: test/e2e/cases/basic.go [hunk 3]
```diff
@@ -31,7 +35,9 @@ func RunBasicCreateTest() {
 	kubeConfigPath := "minKubeconfig"
 	// TODO(nikhiljindal): User should be able to pass gcp project.
 	project, err := runCommand([]string{"gcloud", "config", "get-value", "project"})
-	if err == nil && project == "" {
+	if err != nil {
+		glog.Fatalf("Error getting default project:%v", err)
```
Comment (nikhiljindal) [inline test/e2e/cases/basic.go]: super nit: add a space after ":"

**Comment by nikhiljindal** [comment]
Looks great with minor nits. Will merge this and then rebase and fix nits in https://github.com/GoogleCloudPlatform/k8s-multicluster-ingress/pull/100

**Comment by G-Harmon** [comment]
okay, thanks for fixing up my PR and merging it :)

**Pull request merged**